_examples/apache-zeppelin-example.adoc

```diff
@@ -0,0 +1,102 @@
+:page-link: apache-zeppelin-example
+:page-weight: 100
+:page-labels: [Notebook, Zeppelin, Spark, R, Python, Scala, Java]
+:page-layout: application
+:page-menu_template: menu_tutorial_application.html
+:page-description: This is an example of how to use Apache Zeppelin
+:page-project_links: ["https://github.com/rimolive/zeppelin-openshift", "https://github.com/rimolive/zeppelin-notebooks"]
+
+[[introduction]]
+== Introduction
+
+http://zeppelin.apache.org/[Apache Zeppelin] is an alternative to project http://jupyter.org/[Jupyter] 
+to provide a notebook to run analysis over data and plot graphs in order 
+to show to end users an overview about the data or even doing some exploratory
+analysis and plot some graphs using the data.
+
+This example shows how to use the Zeppelin openShift template created to run on top of
+OpenShift, as well as provide an example notebook to run on top of your OpenShift cluster.
+
+[[architecture]]
+== Architecture
+
+The central piece is the Apache Zeppelin, used to create notebooks. From zeppelin we 
+will connect to the Spark cluster managed by RADAnalytics.io and run some analysis. This
+example will use a pod for the Zeppelin instance and two other pods for the Apache cluster
+(1 master and 1 worker).
+
+[[installation]]
+== Installation
+
+Installing this notebook is straightforward; you simply need to
+create an OpenShift project, deploy a Spark cluster in that project, and
+install and run the notebook image.
+
+=== Prerequisites
+
+Follow the link:/get-started[Get Started] instructions
+to create an OpenShift project with the Oshinko web interface running.
+
+=== Setting up a cluster
+
+From the OpenShift developer console, visit the Oshinko web interface. Use the
+interface to create a new cluster, and take note of what you've called this
+cluster.
+
+=== Setting up the Zeppelin template
+
+In the same project, add the zeppelin template with the following command:
+
+....
+    oc create -f https://radanalytics.io/assets/zeppelin-example/zeppelin-openshift.yaml
+....
+
+After adding the template, create the zeppelin application with the following command:
+
+....
+    oc new-app --template=$namespace/apache-zeppelin-openshift \
+        --param=APPLICATION_NAME=apache-zeppelin \
+        --param=GIT_URI=https://github.com/rimolive/zeppelin-notebooks.git \
+        --param=ZEPPELIN_INTERPRETERS=md
+....
+
+After running the command above, a build will be triggered to add the notebooks hosted 
+in the Github repo, as well as copy the configuration files and install the md interpreter.
+The md interpreter is used to write Markdown blocks in zeppelin.
+
+When the build finishes, it will deploy the newly built zeppelin image. Click on the hostname
+defined in the apache-zeppelin app. In case you don't see a URL for zeppelin, check if a route
+is create with the command:
+
+....
+    oc get route
+....
+
+pass:[<img src="/assets/zeppelin-example/zeppelin1.png" class="img-responsive">]
+
```

Comment:
i love the images!

i think we should move the 2 zeppelin images into the "Usage" section